opcua/server/internal_server.py

```diff
@@ -310,18 +318,45 @@ def close_session(self, delete_subs=True):
 
     def activate_session(self, params):
         self.logger.info("activate session")
+
         result = ua.ActivateSessionResult()
         if self.state != SessionState.Created:
             raise utils.ServiceError(ua.StatusCodes.BadSessionIdInvalid)
+
+        # FIXME: Get the server available UserIdentityToken policy's ids. I don't how to get it in other way.
+        # iserver_policy_ids = ["anonymous", "certificate_basic256sha256", "username"]
+        # Note that it is different from server._policyIDs = ["Anonymous", "Basic256Sha256", "Username"]
```

Comment:
yes that seems to be the correct way to get policy IDs . Just write a comment that we assume that the same polcies are used n all endpoints.. maybe add a FIXME. I am not sure what spec says but in theory it should be possible to have different policies per endpoints